### PR TITLE
Upgrade bundler version

### PR DESCRIPTION
**Pull request by alexlrobertson**
https://app.clubhouse.io/bonusly/story/9648/upgrade-bundler-to-version-2-1-0-or-later-in-bamboo-id

### CODE REVIEW

**Review by ehinson — APPROVED** [review]
This looks good to me. I approve.

File: bamboo_id.gemspec [hunk 1]
```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.14"
+  spec.add_development_dependency 'bundler', '~> 2.2.18'
```
Comment (ajaswa) [inline bamboo_id.gemspec]: Why the change in quotes?

**Review by ajaswa — APPROVED** [review]

File: bamboo_id.gemspec (hunk 1 above)
Comment (alexlrobertson) [reply bamboo_id.gemspec]: This was copied from bonusly/tangolicious#1
